timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-18 09:42:30,ERROR,rahul17,create_or_pass_folder,201,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 194, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:rahul17
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>rahul17</QueryParameterValue></Error>"
2024-06-18 09:42:30,ERROR,rahul17,create_or_pass_folder,210,0,"Function create_or_pass_folder Error creating folder'rahul17': Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:rahul17
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>rahul17</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'rahul17': Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:rahul17
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>rahul17</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 194, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:rahul17
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:045cf0a2-f01e-0005-5535-c18179000000
Time:2024-06-18T04:12:29.7839877Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>rahul17</QueryParameterValue></Error>"
2024-06-18 09:42:30,INFO,rahul17,home,1410,1,User rahul17 logged in successfully,
2024-06-18 09:42:31,INFO,rahul17,update_bar_chart_from_blob,322,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-18 09:42:36,INFO,rahul17,update_bar_chart_from_blob,322,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-18 09:42:39,INFO,rahul17,get_data_source,2333,1,table_update route successfully send data,
2024-06-18 09:42:45,INFO,rahul17,get_data_source,2333,1,table_update route successfully send data,
2024-06-18 09:42:46,INFO,rahul17,update_bar_chart_from_blob,322,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-18 09:42:46,INFO,rahul17,delete_files,2281,1,Selected vault files deleted successfully,
2024-06-18 09:42:47,INFO,rahul17,get_data_source,2333,1,table_update route successfully send data,
2024-06-18 09:42:47,INFO,rahul17,Cogni_button,1725,1,CogniLink load button pressed,
2024-06-18 09:42:48,INFO,rahul17,get_data_source,2333,1,table_update route successfully send data,
2024-06-18 09:43:07,INFO,rahul17,update_bar_chart_from_blob,322,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-18 09:43:10,INFO,rahul17,update_when_file_delete,601,1,Function update_when_file_delete Data Loaded Successfully,
2024-06-18 09:43:10,INFO,rahul17,Cogni_button,1737,1,Load Data Function executed in 22.433557748794556 seconds,
2024-06-18 09:43:13,INFO,rahul17,get_data_source,2333,1,table_update route successfully send data,
2024-06-18 09:46:16,INFO,rahul17,perform_lda____summ,1132,1,Summary topics generated.,
2024-06-18 09:46:16,INFO,rahul17,handle_summary_input,2025,1,Summary Generated in 14.50039553642273 seconds,
